### PR TITLE
[WIP] Applied changes from PR 3828 into the codebase

### PR DESCRIPTION
**Pull request by gmosx**
I manually applied the changes from #3828.
Also some minor changes to make it 'work'.

@ktsaou and @julsemaan, please review!

### CODE REVIEW

File: web/gui/Makefile.am [hunk 1]
```diff
@@ -50,6 +50,7 @@ dist_web_DATA = \
 	demo2.html \
 	demosites.html \
 	demosites2.html \
+    demo-multi-metrics.html \
```
Comment (paulfantom) [inline web/gui/Makefile.am]: nit: wrong indent

**Comment by cakrit** [comment]
We'll implement this in React, after we complete the refactoring.

**Comment by Pingger** [comment]
Any updates on this feature?

**Comment by cakrit** [comment]
@jacekkolasa plz check this one and #3828 and add an issue for this, for the next version of the refactored dashboard.js.

**Pull request closed without merging**